service/storage_service.hh

```diff
@@ -555,6 +556,9 @@ private:
     future<>
     handle_state_replacing_update_pending_ranges(mutable_token_metadata_ptr tmptr, inet_address replacing_node);
 
+    future<> update_quarantined_hosts();
```

Comment:
Please write short descriptions of what these functions do in comments here at the declaration site.